x/ibc/24-validation/validate.go

```diff
@@ -0,0 +1,61 @@
+package validation
```

Comment:
Does this code belong in a separate package? It's pretty trivial but not sure if it should be combined inside 23 package

Reply:
```suggestion
package host
```

Looks good. The dir should be renamed `24-host` as that's the name of the ICS. Imo it's easier that way to review against the spec

Reply:
maybe also add a godoc at the top of the package with a link to the ICS24 spec